cluster-autoscaler/processors/nodegroupset/aws_nodegroups.go

```diff
@@ -21,16 +21,16 @@ import (
 )
 
 // IsAwsNodeInfoSimilar adds AWS specific node labels to the list of ignored labels.
```

Comment:
As you are changing this part of code please also update the comment. It does not capture hat function actually does - that it actually compares nodes for similarity.